src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -18,14 +19,24 @@ export interface ProgressBarProps {
    * @default 'medium'
    */
   size?: Size;
+  /**
+   * Size of progressbar
```

Comment:
Looks like this was copy pasta from above

Reply:
Thanks! Fixed :)